yt/geometry/oct_visitors.pyx

```diff
@@ -373,12 +373,12 @@ cdef class BaseNeighbourVisitor(OctVisitor):
         cdef Oct *neighbour
         cdef bint local_oct
         cdef bint other_oct
-        dx = 1.0 / ((1 << self.oref) << self.level)
+        dx = 1.0 / (self.oref << self.level)
```

Comment:
Now that I read this piece of code... won't it only work if the size ratio between consecutive level is exactly 2?

Reply:
Yes -- *but*, this is an assumption that is already made for the code that is being changed.

Reply:
If we were to fix this, we'd need to do a bit more work on getting the `refinement_factor` code threaded through into Cython.  I am not sure that is appropriate at this point; I believe that we should not expect a strict octree code to ever have something that isn't a factor of 2.  It may be the case that we need to offset this with a multiplicative factor (i.e., if we have one zone refined into four along an axis, we'd want `(2 * self.level)`) but I'm not sure that is necessary at this point.